--- a/llvm/tools/CMakeLists.txt
+++ b/llvm/tools/CMakeLists.txt
@@ -33,23 +33,25 @@ endif()
 # Add LTO, llvm-ar, llvm-config, and llvm-profdata before clang, ExternalProject
 # requires targets specified in DEPENDS to exist before the call to
 # ExternalProject_Add.
-add_llvm_tool_subdirectory(lto)
-add_llvm_tool_subdirectory(gold)
+#add_llvm_tool_subdirectory(lto)
+#add_llvm_tool_subdirectory(gold)
 add_llvm_tool_subdirectory(llvm-ar)
-add_llvm_tool_subdirectory(llvm-config)
-add_llvm_tool_subdirectory(llvm-lto)
-add_llvm_tool_subdirectory(llvm-profdata)
+add_llvm_tool_subdirectory(llvm-as)
+add_llvm_tool_subdirectory(llvm-link)
+#add_llvm_tool_subdirectory(llvm-config)
+#add_llvm_tool_subdirectory(llvm-lto)
+#add_llvm_tool_subdirectory(llvm-profdata)
 
 # Projects supported via LLVM_EXTERNAL_*_SOURCE_DIR need to be explicitly
 # specified.
-add_llvm_external_project(clang)
-add_llvm_external_project(llgo)
-add_llvm_external_project(lld)
-add_llvm_external_project(lldb)
+#add_llvm_external_project(clang)
+#add_llvm_external_project(llgo)
+#add_llvm_external_project(lld)
+#add_llvm_external_project(lldb)
 
 # Automatically add remaining sub-directories containing a 'CMakeLists.txt'
 # file as external projects.
-add_llvm_implicit_projects()
+#add_llvm_implicit_projects()
 
 # Add subprojects specified using LLVM_EXTERNAL_PROJECTS
 foreach(p ${LLVM_EXTERNAL_PROJECTS})
